### DISCUSSION TITLE
Does network connection time out?

### DISCUSSION THREAD

**Discussion by stevempotter** [grblHAL/core#639] [Q&A]
I am using grblHal on a Teensy 4.1 with an ethernet connection to the sender. I am sometimes fiddling with my machine for some minutes before plugging in the ethernet cable and trying to connect to the sender. It seems the ethernet port is dead, no LEDs blinking. I have to power cycle the controller to get ethernet back, a pain if I have already homed and set (0,0,0) carefully with my pendant.

Is it intended behavior that the ethernet port on Teensy gives up after some time? If so, can I disable that?

**Comment by terjeio** [comment]
The network stack was not brought up when link aquired in static IP-mode, [fix committed](https://github.com/grblHAL/iMXRT1062/commit/9bf461a1ce9f0eda2fd67e9fa55aa96c0fe744d2) now.
